README.md

```diff
@@ -0,0 +1,165 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Project Review</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Project Review <a name="about-project"></a>
+
+I created my portfolio using HTML and CSS. 
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. CSS
+
+
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added HTML
+- Added CSS
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- SETUP -->
+### Setup
+
+Clone this repository to your desired folder using:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first.git
```

Comment:
Please enter the correct link to your repository, the one provided leads to a different repository.